spec/appliance_console/logfile_configuration_spec.rb

```diff
@@ -89,22 +90,47 @@
       EOT
     end
 
-    it "stops and starts evm and configures the logfile disk" do
-      subject.new_logrotate_count = 3
-      subject.disk = double(@spec_name, :size => "9999999", :path => "fake disk")
+    context "evm was running" do
```

Comment:
Again I was originally going to code this the way you are requesting. Before posting the PR I consulted http://betterspecs.org/#contexts My interpretation of which suggests the way I have it.